### ISSUE TITLE
Rename PZ2 to "Treasure Chests"

### ISSUE THREAD

**Issue by NilsEnevoldsen** [scryfall/feedback#770]
PZ2 is currently named "You Make the Cube". I think this is a confusing name for two reasons. Firstly, it doesn't contain the cards in the "You Make the Cube" cube (the [Twisted Color Pie Cube](https://scryfall.com/cubes/twisted)). Secondly, it contains cards from current Treasure Chests, which have no real connection to You Make the Cube. I think this set is more recognizably known as "Treasure Chests" or "_Magic Online_ Treasure Chests".

* Wizards calls the PZ2 set symbol the "[_Magic Online_ Treasure Chest symbol](https://magic.wizards.com/en/MTGO/articles/archive/magic-online/treasure-chest-card-list-info)".
* The Magic: The Gathering Wiki calls PZ2 "[Treasure Chests](https://mtg.gamepedia.com/Treasure_Chests)"
* Deckmaster calls PZ2 "[Magic Online Treasure Chest](https://deckmaster.info/set.php?shortcode=PZ2)"

**Comment by silasary** [comment]
Historical reasoning: When first released, PZ2 was the booster pack given out as "You Make The Cube prize packs" (Much like the [Legendary Cube prize packs](https://scryfall.com/sets/pz1)).  

For whatever reason, once _You Make the Cube_ finished, they repurposed the prize boosters as treasure chests.

**Comment by NilsEnevoldsen** [comment]
Yes, even "You Make The Cube prize packs" would be a more accurate name than "You Make The Cube". But I think it's still not very accurate given what the set is today.

With similar justification, I think PZ1 should be renamed "Legendary Cube prize packs".

**Comment by NilsEnevoldsen** [comment]
Relatedly, where is the Legendary Cube on Scryfall? I only see vintage, legacy, modern, and twisted. Am I overlooking it?

**Comment by silasary** [comment]
Nope.  It's currently not listed
